### PR TITLE
Don't show menus datasources

### PR DESCRIPTION
**Pull request by seromenho**

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -58,7 +59,16 @@ function getDataSources() {
   $dataSources = $('#data-sources > tbody');
 
   Fliplet.DataSources.get().then(function (dataSources) {
```
Comment (squallstar) [inline js/interface.js]: didn't we have a `type` filter option?

Comment (seromenho) [inline js/interface.js]: :trollface:

Comment (squallstar) [inline js/interface.js]: just checked, we do. https://github.com/Fliplet/fliplet-api/blob/master/routes/v1/data-sources.js#L78

Comment (seromenho) [inline js/interface.js]: The filter is to choose only menus.
But we can enhance it and check if null then send only null types.

Comment (squallstar) [inline js/interface.js]: `get({ type: 'menu' })`?

Comment (seromenho) [inline js/interface.js]: That works.
But here you want to get `type: null` because "data" data sources have no type.

Comment (squallstar) [inline js/interface.js]: we pass it to sequelize so you can do `{ type: { $notIn: ['menu'] } }`

Comment (squallstar) [inline js/interface.js]: yeah or `type: null`

**Comment by seromenho** [comment]
Fliplet/fliplet-widget-data-source/pull/17

**Pull request closed without merging**